src/Rings/mpoly-ideals.jl

```diff
@@ -0,0 +1,201 @@
+export radical, primary_decomposition, minimal_primes, equidimensional_decomposition_weak,
+          equidimensional_decomposition_radical, equidimensional_hull,
+          equidimensional_hull_radical
+# ideal quotient #######################################################
+@doc Markdown.doc"""
+    quotient(I::MPolyIdeal, J::MPolyIdeal)
+    
+Returns the ideal quotient of `I` by `J`.
+"""
+function quotient(I::MPolyIdeal, J::MPolyIdeal)
+  singular_assure(I)
+  singular_assure(J)
+  return MPolyIdeal(I.gens.Ox, Singular.quotient(I.gens.S, J.gens.S))
```

Comment:
@wdecker by the way, if you check here on the website, you can see which code is not covered by any tests - e.g. this function isn't